lib/regex.js

```diff
@@ -1,34 +1,22 @@
 'use strict';
-var forEach = require('lodash').forEach;
 
+function join(array, joiner) {
+  return array
+    .map(function(val) {
+      return (val || '').trim();
```

Comment:
oh because of https://github.com/stevemao/conventional-commits-parser/blob/master/test/regex.spec.js#L25